app/src/main/java/net/gsantner/markor/activity/DocumentShareIntoFragment.java

```diff
@@ -14,7 +14,6 @@
 import android.content.res.ColorStateList;
```

Comment:
Added a major feature: We can now share a file into markor. This shared file can be:

1. Attached to a note (file will be copied into notebook as usual)
2. Saved to a folder

We can also share a file from markor into markor. This lets us attach the current document to another one.